starter-code/scripts/articleView.js

```diff
@@ -32,58 +35,62 @@ articleView.handleAuthorFilter = function() {
 };
 
 articleView.handleCategoryFilter = function() {
-  /* TODO: Just like we do for #author-filter above, we should handle change
+  /* TODONE: Just like we do for #author-filter above, we should handle change
             events on the #category-filter element. Be sure to reset
             the #author-filter while you're at it! */
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      $('article[data-category="' + $(this).val() + '"]').fadeIn();
+    } else {
+      $('article').not('.template').fadeIn();
+    }
+    $('#author-filter').val('');
+  });
 };
 
 articleView.handleMainNav = function() {
-  /* TODO: Complete the delegated event handler below to help
+  /* TODONE: Complete the delegated event handler below to help
             power the tabs feature.
       Clicking any .tab element should:
         1. Hide all the .tab-content sections.
         2. Fade in the single .tab-content section that is associated with
             the clicked .tab element's data-content attribute. */
-  $('.main-nav').on(/* CODE GOES HERE */);
+  $('.main-nav').on('click', '.tab', function(){
+    $('.tab-content').hide();
+    $('#' + $(this).data('content')).fadeIn();
+  });
```

Comment:
Great job hiding all of the tab-content sections and fading in only the selected one